pkg/iter/entry_iterator_test.go

```diff
@@ -735,6 +735,25 @@ func BenchmarkSortIterator(b *testing.B) {
 		}
 	})
 
+	b.Run("merge sort dedupe", func(b *testing.B) {
+		b.ReportAllocs()
+		b.ResetTimer()
+		for i := 0; i < b.N; i++ {
+			b.StopTimer()
+			var itrs []EntryIterator
+			for i := 0; i < streamsCount; i++ {
+				itrs = append(itrs, NewStreamIterator(streams[i]))
+				itrs = append(itrs, NewStreamIterator(streams[i]))
```

Comment:
double checking: was this line duplicated by accident?

Reply:
I think it's intentional as the test case is related to deduping